### DISCUSSION TITLE
新人也要和群管理谈恋爱！

### DISCUSSION THREAD

**Discussion by This-is-XiaoDeng** [ITCraftDevelopmentTeam/Forum#58] [General]
- 原作: fzzkill
- 更新日期: 2024年06月02日
-----------

时间过得真快，一晃就是三年
前几天，Ball给我发了消息，和我说这公园见面之类的话。我先是愣了些许，没想到她还记得我。
Ball是我的高中同学，依稀记得是我们班的班花。刚入学的时候，也许是机缘巧合，我坐在了她旁边。这一坐就是三年，渐渐的，我有些喜欢上她了。她似乎不太喜欢我，甚至可以用讨厌形容。还记得她和我说过得一句话：“别再粘着我了，我不想和你待太久。”话虽如此，她却没有一次想要换过座位。
毕业之后，她以700的高分考入江北大学，可我却没考好，也没能和她在一所大学待着。最开始毕业分离的时候，也许是恨自己恨铁不成钢，没能考入江北大学，并没有想她，每天都沉浸在后悔之中。慢慢的，时间冲淡了这些事情，也就一心到学业只中了。
收拾东西。到了公园，看见了一个熟悉的身影：淡蓝色上衣，披着散发，挎着包，似乎在寻找着什么。我知道那是Ball，可我终究是不敢认的。她先是回头，看见了我，冲我笑了一下。
她笑的很甜，仿佛融化了我的心。脸有些红，一时间竟忘了怎么说话。她拉着我的手，摇晃着。
“好……好久不见”我有些害羞，一个字一个字蹦出来的。
“是呢，好久不见”她说着，声音很甜，似乎融化了我的心。拉着她，进入了公园。
公园的风景固然美丽，可是在Ball面前，一切都不值一提。她还是那样的好看。在一座桥上，Ball在看着河流流过，看着河岸的石头，看水中的波纹，看水里的草，我什么也没看，一直在看着Ball。
下午过得还是太快了，一会就到晚上了。过得很快，她在我心里跑的也很快。
还是那座桥，还是同样的两个人，不一样的是，她两只手都扶在了我的肩上。
“[你的名字]我喜欢你很久了，这三年也想你很久了，所以……做我男朋友吧，不能拒绝哦。”她冲我说着，脸有些红。
见我迟迟没有反应，她先是犹豫了一下，随后亲了上去。“没说话，就当你同意了啊！”
这一吻，融化了我的心。

-----------

早上，我还是出门转，却是遇见了Ball。
见着Ball是和xxtg在一起的，他们说的不亦说乎，我有些生气，却又不敢过去。
Ball似乎看见了我，向我跑了过来。我没有迎上，而是在此地没有走动。
“早上好啊”Ball和我说着。便拉着我的手想去xxtg那里。我是甩开了她的手的，脸色有些生气，心里是有些酸的。
“你心情不好吗？”她关心着我。没有应答，也只是在那里站着，“发生什么事了？”她补充道。
“生气了。”我说着，眼神逃避着。
“为什么啊？”她追问道。停顿，她却笑了出来，“你小子是不是以为我喜欢上别人了？没有的事情啊！”说罢，拉着我的手。我是半信半疑的，却还是让她拉着走了。
至于xxtg，我是认识他的。他便是我们社团社长，又是闻名的大学霸。
xxtg看我这幅模样，不禁笑出了声，“醋王别吃醋了！”我没有理会，只是紧紧的握着Ball的手，生怕她被人拐走了。
xxtg打趣着我。“[你的名字]就这多愁善感，跟个林黛玉一样，貌似是娘胎里投错性别了……便是这样，和他好的只要和别人好了，他就有点吃醋了。”“你也多安慰安慰他吧”xxtg转过去，祝福着Ball，我没看见Ball干了什么，只是知道她搂住我了。
“好！那么人都齐了，我们就出发吧。”说罢，却是让Ball打住了。“你要和我们出去玩会吗？”她问着我。
xxtg突然想到了什么——昨天摇人打算今天出去Ctrl却是把我忘记了。他的神色有点慌张。
“好……好吧”看在Ball，便是同意了。说罢，她拉着我的手。
“那就出发吧！”xxtg说着。貌似是忘记了人，忘记了谁呢？
[未完待续]

**Comment by zhao6888** [comment]
阳光透过树梢，斑驳地洒在公园的小道上，我们四个人就这样肩并肩地走着。xxtg似乎已经忘记了昨天的尴尬，一路上谈笑风生，和Ball不时交换着眼神，而我则默默地跟在他们身后，心情复杂。

我们来到了公园的湖边，湖水清澈见底，偶尔有几条小鱼跃出水面，激起一圈圈涟漪。Ball突然停下脚步，转过身看着我，眼中闪烁着光芒。

"[你的名字]，我知道你心里在想什么，"她认真地说，"我和xxtg只是朋友，我们之间没有别的。你相信我吗？"

我看着她的眼睛，那里有我熟悉的真诚和温暖，心中的疑虑慢慢消散。我点了点头，"我相信你，Ball。"

她露出了一个释然的微笑，然后拉着我的手，我们一起走向湖边的长椅坐下。xxtg和另一个朋友也跟了过来，四个人围坐在一起，开始分享各自的趣事和近况。

时间在欢声笑语中悄悄流逝，太阳慢慢爬上了头顶，我们决定去附近的餐厅吃午饭。xxtg提议去一家他经常光顾的日式料理店，据说那里的寿司和刺身都非常美味。

我们穿过熙熙攘攘的街道，来到了那家日式料理店。店内装潢简洁而雅致，服务员热情地迎接我们，带我们到了一个靠窗的位置。

我们点了一桌子的菜，有新鲜的刺身、美味的寿司、香脆的天妇罗，还有热气腾腾的拉面。大家边吃边聊，气氛轻松愉快。

吃完饭后，我们决定去附近的电影院看一场电影。xxtg提前订好了票，是一部最近很火的爱情喜剧。我们买了爆米花和饮料，找到了自己的座位。

电影开始了，是一部轻松幽默的爱情故事，我们不时发出会心的笑声。我偷偷看了一眼Ball，她也正看着我，我们相视一笑，仿佛所有的误会和不安都烟消云散了。

电影结束后，我们走出电影院，夜幕已经降临。xxtg提议去附近的酒吧坐坐，放松一下。我和Ball互相看了一眼，点了点头。

我们来到了一家风格独特的酒吧，昏暗的灯光下，乐队正在演奏着柔和的爵士乐。我们找了个角落的位置坐下，点了鸡尾酒和小吃。

在柔和的音乐和微醺的氛围中，我们聊起了过去和未来，谈笑风生。我看着身边的Ball，心中充满了感激和幸福。也许，这就是青春最美好的模样。

夜深了，我们告别了xxtg和另一个朋友，我和Ball手牵手走在回家的路上。月光洒在她的脸上，她的笑容在夜色中显得格外温柔。

"[你的名字]，今天真的很开心，"她轻声说，"有你在我身边，我觉得好幸福。"

我紧紧握住她的手，"我也是，Ball，有你真好。"

我们就这样慢慢走着，月光下的影子拉得很长很长，仿佛预示着我们未来的路，也会这样手牵手，一直走下去。

[未完待续]

**Comment by xxtg666** [comment]
不是fzz写的我不看！

**Comment by zhao6888** [comment]
原告：xxtg，ball

**Comment by ghost** [comment]
?????
